portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro

```diff
@@ -2356,16 +2356,22 @@ definition {
 
 		SelectFrame(locator1 = "IFrame#MODAL_BODY");
 
-		if (${dmFolderName} == "Home") {
-			if (IsElementPresent(key_breadcrumbName = "Home", locator1 = "BreadcrumbPortlet#BREADCRUMB_FOLDER")) {
+		echo("## * dmFolderName: ${dmFolderName}");
```

Comment:
@AliciaGarciaGarcia, shouldn't we remove the echos from the tests?

Reply:
This could be helpful when debugging a test failure